api/src/main/java/org/apache/cloudstack/api/command/user/loadbalancer/CreateLoadBalancerRuleCmd.java

```diff
@@ -114,7 +114,7 @@
         + "rule will be created for. Required when public Ip address is not associated with any Guest network yet (VPC case)")
     private Long networkId;
 
-    @Parameter(name = ApiConstants.PROTOCOL, type = CommandType.STRING, description = "The protocol for the LB such as tcp, udp or tcp-proxy.")
+    @Parameter(name = ApiConstants.PROTOCOL, type = CommandType.STRING, description = "The protocol for the LB such as tcp or tcp-proxy.")
     private String lbProtocol;
```

Comment:
@davidjumani might 'udp' be supportd by some lb hardware devices ?

Reply:
@weizhouapache good point, I'll move the check to the NetworkHelperImpl.java